### PR TITLE
Add armor62 encoding to sign/verify

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco @oconnor663

### CODE REVIEW

File: go/saltpack/const.go [hunk 1]
```diff
@@ -39,6 +39,22 @@ const EncryptionArmorHeader = "BEGIN KEYBASE ENCRYPTED MESSAGE"
 // armored KB message.
 const EncryptionArmorFooter = "END KEYBASE ENCRYPTED MESSAGE"
 
+// SignedArmorHeader is the header that marks the start of signed
+// armored KB message.
+const SignedArmorHeader = "BEGIN KEYBASE SIGNED MESSAGE"
```
Comment (maxtaco) [inline go/saltpack/const.go]: May as well change it over to the right text:

`BEGIN KEYBASE SALTPACK SIGNED MESSAGE`

Comment (maxtaco) [inline go/saltpack/const.go]: BTW, does that look gross?  It seems like too much stuff... hmmm....

`Begin Keybase SaltPack signed message.`

Would that be better?

Comment (patrickxb) [inline go/saltpack/const.go]: I prefer mixed case.

BTW, these should probably be added to the spec as there is no way to know the "right" text.

**Comment by patrickxb** [comment]
I changed the header/footer.  PTAL.

**Comment by maxtaco** [comment]
:+1:

**Pull request merged**